core/test/unit/cdk-nag/nag-data-domain.test.ts

```diff
@@ -86,15 +87,103 @@ NagSuppressions.addResourceSuppressionsByPath(
 
 NagSuppressions.addResourceSuppressionsByPath(
   dataDomainStack,
-  'dataDomain/myDataDomain/DataDomainWorkflow',
-  [{ id: 'AwsSolutions-SF2', reason: 'DataDomainWorflow construct not in the scope of this NAG test' }],
+  'dataDomain/s3-access-logs/Resource',
+  [{ id: 'AwsSolutions-S1', reason: 'The S3 bucket used for access logs can\'t have access log enabled' }],
+);
+
+NagSuppressions.addResourceSuppressionsByPath(
+  dataDomainStack,
+  'dataDomain/myDataDomain/nracWorkflow/nracStateMachine/Resource',
+  [{ id: 'AwsSolutions-SF2', reason: 'X-ray not required for the CentralGovernance workflow' }],
+);
+
+NagSuppressions.addResourceSuppressionsByPath(
+  dataDomainStack,
+  'dataDomain/myDataDomain/tbacWorkflow/tbacStateMachine/Resource',
+  [{ id: 'AwsSolutions-SF2', reason: 'The Step Function doesn\'t need X-ray' }],
   true,
 );
 
 NagSuppressions.addResourceSuppressionsByPath(
   dataDomainStack,
-  'dataDomain/s3-access-logs/Resource',
-  [{ id: 'AwsSolutions-S1', reason: 'The S3 bucket used for access logs can\'t have access log enabled' }],
+  'dataDomain/CdkLfAdmin/LogRetentionLambdaExecutionRoleCdkLfAdminlfAdminCreateFn/DefaultPolicy/Resource',
```

Comment:
If entire CdkLfAdmin resource including its children can be skipped because they are not in the scope, I suggest to rewrite a single test like this:
```
NagSuppressions.addResourceSuppressionsByPath(
  dataDomainCrawlerStack,
    'dataDomain/CdkLfAdmin',
    [
      { id: 'AwsSolutions-S1', reason: 'Not the purpose of this NAG to test LakeFormationAdmin construct' },
      { id: 'AwsSolutions-L1', reason: 'Not the purpose of this NAG to test LakeFormationAdmin construct' },
      { id: 'AwsSolutions-IAM4', reason: 'Not the purpose of this NAG to LakeFormationAdmin construct' },
      { id: 'AwsSolutions-IAM5', reason: 'Not the purpose of this NAG to LakeFormationAdmin construct' }
    ],
    true,
  );
```

Reply:
it would replace the next 6 nag suppressions